cli/pcluster/config/pcluster_config.py

```diff
@@ -385,31 +381,34 @@ def __check_account_capacity(self):  # noqa: C901
 
             # Test Master Instance Configuration
             self.__ec2_run_instance(
-                max_size,
+                1,
                 InstanceType=master_instance_type,
                 MinCount=1,
                 MaxCount=1,
                 ImageId=latest_alinux_ami_id,
                 SubnetId=master_subnet,
+                SecurityGroupIds=security_groups_ids,
                 CpuOptions=master_cpu_options,
                 Placement=master_placement_group,
                 DryRun=True,
             )
 
             # Test Compute Instances Configuration
             self.__ec2_run_instance(
-                max_size,
```

Comment:
Since we're always passing 1 to the `max_size` arg of `__ec2_run_instance`, I think we should get rid of that arg.

Reply:
Also, it would be nice to include a comment explaining that the number of instances is hardcoded because we can't check for capacity this way anymore.

Reply:
Done